Sources/Schema.swift

```diff
@@ -28,13 +28,6 @@ enum SchemaBuilder: Builder {
     case boolean(metadata: MetadataBuilder)
 
     public init(map: Map) throws {
```

Comment:
I realize this wasn't modified by your PR but builders and their initializers should be internal. IDK why it didn't complain that we have a public initializer on an internal type.